cms/djangoapps/contentstore/views/videos.py

```diff
@@ -11,7 +11,7 @@
 from contextlib import closing
 from datetime import datetime, timedelta
 from uuid import uuid4
-
+from boto.s3.connection import S3Connection
```

Comment:
via `django-pyfs` it was importing on top. So I am maintaining it that way to avoid any surprise breakage.